CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 ## Unreleased
 
+* Fix typo from PR #3944 ([PR #3945](https://github.com/alphagov/govuk_publishing_components/pull/3945))
 * Improve test coverage of contextual breadcrumb logic ([PR #3944](https://github.com/alphagov/govuk_publishing_components/pull/3944))
```

Comment:
Maybe you can just add the PR link to the second line instead of creating a new line?

Reply:
Oh nice idea, thanks will do that 👍